test/recipes/80-test_ssl_new.t

```diff
@@ -56,6 +56,7 @@ my $no_ocsp = disabled("ocsp");
 my %conf_dependent_tests = (
   "02-protocol-version.conf" => !$is_default_tls,
   "04-client_auth.conf" => !$is_default_tls,
+  "05-sni.conf" => $no_tls || disabled("tls1_1"),
```

Comment:
In any case if we have no_tls then disabled("tls1_1") will always be true, so it should be sufficient just to test for the latter.